src/main/java/com/google/devtools/build/lib/rules/cpp/CppOptions.java

```diff
@@ -244,6 +244,20 @@ public String getTypeDescription() {
   )
   public DynamicMode dynamicMode;
 
+  @Option(
+    name = "rename_dll",
```

Comment:
After some discussion with our C++ rules owner. We want to enable this behaviour by default in future. However, it should be an incompatible change for Bazel, so it has to follow our [incompatible change policy](https://docs.bazel.build/versions/master/backward-compatibility.html#incompatible-changes-and-migration-recipes)
Can you rename the flag to maybe `--incompatible_avoid_conflict_dlls`? And please file another issue to track this incompatible change. Thanks!